### DISCUSSION TITLE
Combine options of Datetime Display

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3734] [Feature Requests]
Perhaps we should combine these formatting options of the datetime Display?

![Screen Shot 2020-11-11 at 3 53 43 PM](https://user-images.githubusercontent.com/522079/104963903-73ad1480-59a9-11eb-9433-9cb2a0c20c4f.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
